CMakeLists.txt

```diff
@@ -2,5 +2,6 @@
 # executed from 'project/build' with 'cmake ../'.
 cmake_minimum_required(VERSION 2.6)
 find_package(Rock)
+add_definitions(-fext-numeric-literals)
```

Comment:
Setting the flag **explicitly** gets obsolete, when we make sure that c++11 and gnu++11 are not both set -- see issues on rock-core/base-cmake

> can't we change the code to not use the boost macro for initialization ?

What macro are you now referring to exactly?

> (which should be tested before being added btw.)
testing led to the identification of the gnu+11 vs. c++11 issue -- setting the flag would just make sure it builds in any case